Tests/SentryTests/TestUtils/Invocations.swift

```diff
@@ -4,44 +4,44 @@ import Foundation
  * For recording invocations of methods in a list in a thread safe manner.
  */
 class Invocations<T> {
-    
```

Comment:
Again please extra PR for such changes.

Reply:
👍🏻 https://github.com/getsentry/sentry-cocoa/pull/2173